src/nl/rubensten/texifyidea/util/Psi.kt

```diff
@@ -398,3 +398,32 @@ fun PsiFile.hasBibliography() = this.commandsInFileSet().any { it.name == "\\bib
  * @return `true` when the fileset has a bibliography included, `false` otherwise.
  */
 fun PsiFile.usesBiber() = this.commandsInFileSet().any { it.name == "\\printbibliography"}
+
+private fun splitContent(element: PsiElement, delimiter: String = ",") = element.firstChildOfType(LatexNormalText::class)?.text?.split(delimiter)
```

Comment:
Also, `splitContent` can return `null`? Is this supposed to happen, or is an empty list better suited?

When writing public util functions I think it's is generally a good idea to explicitly specify the return type.